assembly/std/__tests__/context.spec.ts

```diff
@@ -33,4 +34,8 @@ describe('Context', () => {
     const coins = transferredCoins();
     expect(coins).toBe(0);
   });
+
+  test('isDeployingContract', () => {
+    expect(isDeployingContract()).toBe(true);
```

Comment:
Why this test is true for tests ?

Reply:
I dont quite understand - should it return anything else than true in tests where we are using mocked vm ?

Reply:
let's discuss that @BatiGencho, it will be simpler

Reply:
already discussed